.circleci/config.yml

```diff
@@ -876,23 +876,6 @@ build-and-deploy-with-context-defaults: &build-and-deploy-with-context-defaults
 
 workflows:
   version: 2
-  scheduled-testing:
```

Comment:
Was this removed on purpose?

Reply:
It looks like this happened because we branched off of staging. @mmarcotte can you weigh in on if this commit belongs in test or not?

Reply:
fwiw, I've seen this frequently. It might be less confusing if, when merging anything to staging, staging is immediately back-merged to test.

Reply:
my bad. i did a bit of merging yesterday, and I missed this after updating the scheduled testing. should be up to date now.